src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure implements Area, ColorCreator {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double countArea() {
+        return Math.PI * (radius * radius);
+    }
+
+    @Override
+    public Color getColor() {
```

Comment:
this method should return value of color field and not generte random color each

Reply:
Changed